packages/server-core/src/media/file-browser/file-browser.class.ts

```diff
@@ -132,12 +136,15 @@ export class FileBrowserService
 
     if (params.provider && !isAdmin) {
       const knexClient: Knex = this.app.get('knexClient')
-      const projectPermissions = await knexClient
-        .from(projectPermissionPath)
-        .join('project', `${projectPermissionPath}.projectId`, 'project.id')
-        .where(`${projectPermissionPath}.userId`, params.user!.id)
-        .select()
-        .options({ nestTables: true })
+      const projectPermissions: { 'project-permission': ProjectPermissionType; project: ProjectType }[] =
+        await knexClient
+          .from(projectPermissionPath)
+          .join(projectPath, `${projectPermissionPath}.projectId`, `${projectPath}.id`)
+          .where(`${projectPermissionPath}.userId`, params.user!.id)
+          .select()
+          .options({ nestTables: true })
+
+      total = projectPermissions.length
```

Comment:
Although this solves the bug for 'projects/' directory, this will return incorrect total for subdirectories. Moving this line inside `if (directory === 'projects/')` should fix this problem but I'm not sure if this is the desired way to approach this. @HexaField what do you think?

Reply:
This seems like a good solution. As long as we aren't using git every time we need to get the permissions.